plugins/netmotion.yaml

```diff
@@ -0,0 +1,106 @@
+# Plugin Info
+version: 0.0.1
+title: Netmotion
+description: Log parser for Netmotion Syslog
+parameters:
+  - name: listen_address
+    label: Listen Address
+    description: A TCP network address of the form `<ip>:<port>`
+    type: string
+    default: "0.0.0.0:514"
+
+# Set Defaults
+# {{$listen_address := default "0.0.0.0:514" .listen_address}}
+
+# Pipeline Template
+pipeline:
+  - id: netmotion_input
+    type: tcp_input
+    listen_address: {{ $listen_address }}
+    labels:
+      log_type: netmotion
+      plugin_id: {{ .id }}
+    output: handle_new_lines
+
+  # tcp input uses \n to break messages into seperate
+  # entries, this can be a problem for Syslog if the
+  # structured_data key=value pairs or message fields contain
+  # a \n. This is unlikly, but not impossible. It has been
+  # observed with rfc5424 structured_data field.
+  # '<' can be relied on to be the first character of every
+  # syslog message for both rfc 5424 and 3164
+  - id: handle_new_lines
+    type: recombine
+    combine_field: $record
+    is_first_entry: "$record startsWith '<'"
+    output: m_user_router
+
```

Comment:
Add description explaining this handles brackets for m_user key/value pairs within structured data